src/__test__/components/plots-and-tables/ViolinPlotIndex.test.jsx

```diff
@@ -105,7 +106,7 @@ describe('ViolinIndex', () => {
   it('loads by default the gene with the highest dispersion, allows another to be selected, and updates the plot\'s title', async () => {
     await renderViolinIndex();
 
-    const geneSelection = rtl.screen.getAllByRole('tab')[0];
+    const geneSelection = rtl.screen.getAllByText('Gene Selection')[0];
     expect(geneSelection).toHaveTextContent('Gene Selection');
```

Comment:
This test doesn't do anything. It queries for an element containing the text, and expects the retrieved element to have the text, so if the element is there, of course expects will be true.

If you're only looking for 1 element, you can just use `getByText('Gene Selection')`.

The replaced test checks for the order of the element. If this is not what we're testing for, instead of `toHaveTextContent`, you can just check for the existence of the element using something like `expect(geneSelection.length).toEqual(1)`